### PR TITLE
Arabic hebrew translations

### LINKED ISSUE #108: Arabic Hebrew Json files
Write translations for both Arabic and Hebrew in 2 separate JSON files for everything in the app other than the data from the database

### PR DESCRIPTION
**Pull request by Karyum**
close #108

### CODE REVIEW

File: src/app-languages/arabic.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "/careers/details": {
+    "salary": {
+      "title": "راتب",
+      "junior": "مبتدئ",
+      "senior": "خبير"
+    },
+    "grades": {
+      "typeOfGrades": "علامات",
+      "yours": "علامتك",
```
Comment (shiryz) [inline src/app-languages/arabic.json]: Is more than one grade?
علاماتك؟

File: src/app-languages/arabic.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      "required": "مطلوب",
+      "bagrut": "بجروت",
+      "tawjihi": "توجيهي",
+      "psychometric": "بسيخومتري"
+    }
+  },
+  "navbar": {
+    "editGrades": "تغيير العلامات",
```
Comment (shiryz) [inline src/app-languages/arabic.json]: تعديل*

File: src/app-languages/hebrew.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "continueWithFacebook": "המשיך עם פייסבוק",
```
Comment (shiryz) [inline src/app-languages/hebrew.json]: המשך*

File: src/app-languages/hebrew.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      "typeOfGrades": "סוג הציון",
+      "yours": "הציונים שלך",
+      "required": "נדרש",
+      "bagrut": "בגרות",
+      "tawjihi": "תאוגיהי",
+      "psychometric": "פסיכומטרי"
+    }
+  },
+  "navbar": {
+    "editGrades": "לערוך ציונים",
```
Comment (shiryz) [inline src/app-languages/hebrew.json]: עריכת ציונים*

**Comment by Karyum** [comment]
@shiryz bless you :pray: thanks

File: src/app-languages/arabic.json (hunk 1 above)
Comment (shiryz) [reply src/app-languages/arabic.json]: Then the hebrew one needs to be changed

File: src/app-languages/hebrew.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "/careers/details": {
+    "salary": {
+      "title": "שכר",
+      "junior": "בוגר טרי",
+      "senior": "בכיר"
+    },
+    "grades": {
+      "typeOfGrades": "סוג הציון",
+      "yours": "הציונים שלך",
```
Comment (shiryz) [inline src/app-languages/hebrew.json]: *ציון

**Pull request merged**